src/lib.rs

```diff
@@ -169,6 +170,34 @@ impl Device {
             err => panic!("unknown get_write_chunksize retval {:?}", err),
         }
     }
+
+    pub fn read_pins(&mut self) -> Result<u8> {
+        let mut pins : u8 = 0;
+        let pins_ptr = std::slice::from_mut(&mut pins).as_mut_ptr();
+
+        let result = unsafe { ffi::ftdi_read_pins(self.context, pins_ptr) };
+
+        match result {
+            0 => Ok(pins),
+            -1 => Err(Error::RequestFailed), // read pins failed
+            -2 => unreachable!("uninitialized context"),
+            _ => Err(Error::unknown(self.context)),
+        }
+    }
+
+    pub fn set_baudrate(&mut self, baudrate : u32) -> Result<()> {
```

Comment:
All our API needs docs, but this function probably the most for now. Does it include the "but the actual baud rate is 16 times higher" property in bitbang, assuming we implement it sooner or later (and do we want it to)? What accuracy does it guarantee? Can we somehow get back the actual baud rate configured? What happens to the baud rate what we switch modes once we support that?